### ISSUE TITLE
add webhook for new game

### ISSUE THREAD

**Issue by mysterious-static** [velinath/increasingly-nervous-man#3]
vf could call and post link in channel

**Comment by mysterious-static** [comment]
366737195744100352

use sqs

**Comment by mysterious-static** [comment]
```node
var AWS = require('aws-sdk');
AWS.config.update({region: process.env.region}); //implemented in heroku
var sqs = new AWS.SQS({apiVersion: '2012-11-05'});

var queueUrl = process.env.sqs_queue_url; //Implemented in Heroku

var params = {
  AttributeNames: [
    "SentTimestamp"
  ],
  MaxNumberOfMessages: 1,
  MessageAttributeNames: [
    "All"
  ],
  QueueUrl: queueUrl,
  VisibilityTimeout: 0,
  WaitTimeSeconds: 0
};

sqs.receiveMessage(params, function(err, data) {
  if (err) {
    console.log("Receive Error", err);
  } else if (data.Messages) {
    var message = data.Messages[0];
    var channel = client.channels.get('366737195744100352');
    channel.send(message.Moderator + ' has opened ' + message.GameTitle + '. Thread Link: https://forums.somethingawful.com/showthread.php?threadid=' + message.threadId);
    var deleteParams = {
      QueueUrl: queueUrl,
      ReceiptHandle: message.ReceiptHandle
    };
    sqs.deleteMessage(deleteParams, function(err, data) {
      if (err) {
        console.log("Delete Error", err);
      } else {
        console.log("Message Deleted", data);
      }
    });
  }
});
```

**Comment by mysterious-static** [comment]
closed in f92af0d
